lib/govuk_message_queue_consumer/rabbitmq_config.rb

```diff
@@ -10,7 +10,24 @@ def self.from_environment(env)
         user: fetch(env, "RABBITMQ_USER"),
         pass: fetch(env, "RABBITMQ_PASSWORD"),
         recover_from_connection_close: true,
-      }
+      }.merge(tls_config(env))
```

Comment:
You can construct a URL in this form which Bunny will accept:

`amqp(s)://(username):(password)@(host):(port)` 

- which is how it works on Publishing API. But you still have to [supply the cert/key/verify/etc variables separately](https://github.com/ruby-amqp/bunny/blob/main/docs/guides/tls.md#providing-certificates--keys-when-using-amqps-uris)

Reply:
Suggestion - how about we go with this as-is for now, and put a separate task & PR in for refactoring this to allow `RABBITMQ_URL` ?

Reply:
Enthusiastic +1 from Replatforming for `RABBITMQ_URL`. Having to deal with lots of different environment variables is nasty and error-prone.

Reply:
I'd actually planned to get rid of those snowflakey GOV.UK env vars: https://github.com/alphagov/govuk_message_queue_consumer/pull/80

Also, strong preference for using files for the certs/keys rather than environment variables. Using env vars for such things makes it harder to do privilege separation well, besides cluttering up the environment with very long strings which it's not really designed for.

Reply:
Lemme know if you wanna work together on sorting out this config interface! :)

Reply:
Hey thanks a lot for starting PR #80! Happy to work together on it - we were planning to do this ourselves after trying out what we have on integration and using that to figure out what we actually need, so it's great that you were already thinking the same way. 

I think we're all agreed now that `RABBITMQ_URL` (or possibly some slightly different name) is the direction we should be heading, and in principle that should give us everything we need. So let's do that. 

I suggest sticking with `RABBITMQ_URL` as the env var name, as it's more-or-less a de-facto standard across multiple Rabbit clients. I could be persuaded otherwise if anyone else has strong feelings though.

Reply:
So when PR #80 is ready, if we're all happy with it, let's accept that one and close this one as unmerged